pkg/controllers/counter/controller.go

```diff
@@ -16,13 +16,12 @@ package counter
 
 import (
 	"context"
-	"net/http"
 	"time"
 
 	"k8s.io/apimachinery/pkg/util/sets"
 
 	"github.com/aws/karpenter-core/pkg/controllers/state"
-	operatorcontroller "github.com/aws/karpenter-core/pkg/operator/controller"
+	corecontroller "github.com/aws/karpenter-core/pkg/operator/controller"
```

Comment:
Don't see a reason to alias
```suggestion
	"github.com/aws/karpenter-core/pkg/operator/controller"
```

Reply:
Controller is already used from "sigs.k8s.io/controller-runtime/pkg/controller"